packages/ERTP/src/types.js

```diff
@@ -260,55 +277,17 @@
  */
 
 /**
- * @callback MakeIssuerKit
- * @param {string} allegedName
- * @param {AssetKind} [assetKind=AssetKind.NAT]
- * @param {AdditionalDisplayInfo} [displayInfo={}]
- * @param {ShutdownWithFailure=} optShutdownWithFailure If this issuer fails
- * in the middle of an atomic action (which btw should never happen), it
- * potentially leaves its ledger in a corrupted state. If this function was
- * provided, then it the failed atomic action will call it, so that some
- * larger unit of computation, like the enclosing vat, can be shutdown
- * before anything else is corrupted by that corrupted state.
- * See https://github.com/Agoric/agoric-sdk/issues/3434
- * @returns {IssuerKit}
- *
- * The allegedName becomes part of the brand in asset descriptions. The
- * allegedName doesn't have to be a string, but it will only be used for
- * its value. The allegedName is useful for debugging and double-checking
- * assumptions, but should not be trusted.
- *
- * The assetKind will be used to import a specific mathHelpers
- * from the mathHelpers library. For example, natMathHelpers, the
- * default, is used for basic fungible tokens.
- *
- *  `displayInfo` gives information to UI on how to display the amount.
- *
- * @typedef {Object} IssuerKit
```

Comment:
As noted above, please leave this `IssuerKit` type spelled out for now.

I'm all for moving the `MakeIssuerKit` stuff by the function code.

Reply:
porque? `makeIssuerKit` is the only thing that can make one. It's like a class constructor. Putting it on the return type means that when you are looking at an `IssuerKit` and want to figure out how it gets generated you can command-clik to get to the maker/constructor.

Reply:
My concern here is mis-placed. `Issuer` is the thing that is interface-like that's important to keep spelled out.